### PR TITLE
CLOUD-2272 Modify DX Kafka Module

### PR DESCRIPTION
**Pull request by vibindaniel**
# Description

Refactor Kafka Module

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change
(fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration

- [ ] Sanity Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: modules/apps/ui/pre_run.sh
Comment (zumic96) [inline modules/apps/ui/pre_run.sh:1]: This is leftover from moving it to scripts folder, same for cron and handler?

File: modules/apps/ui/main.tf [hunk 1]
```diff
@@ -31,19 +33,19 @@ locals {
         "oauth2-proxy" = local.is_proxy_auth_enabled ? {
           enabled = true
           config = {
-            tenantId     = local.auth_secret["AUTH__TenantId"]
-            clientId     = local.auth_secret["AUTH__ClientId"]
+            tenantId     = local.auth_secret["aUTH__TenantId"]
```
Comment (zumic96) [inline modules/apps/ui/main.tf:36]: Caps lock typo?
Same on API.

**Review by zumic96 — CHANGES_REQUESTED** [review]

File: modules/common/kafka/outputs.tf [hunk 2]
```diff
@@ -1,22 +1,21 @@
 output "aws_secrets" {
-  value       = [{ name = "${var.app_name}-kafka-creds" }]
+  value = [
+    { name = "${replace(local.service_account, local.prefix, "")}-kafka-creds", type = "kafka" },
```
[the remark this answers is not in the document]
Comment (vibindaniel) [reply modules/common/kafka/outputs.tf:3]: This is created in the ix-kafka-users-topic repo

File: modules/apps/ui/pre_run.sh
Comment (vibindaniel) [reply modules/apps/ui/pre_run.sh:1]: deleted

File: modules/apps/ui/main.tf (hunk 1 above)
Comment (vibindaniel) [reply modules/apps/ui/main.tf]: fixed

**Review by zumic96 — APPROVED** [review]

**Pull request merged**